packages/admin-ui/src/table/components/table-cell.tsx

```diff
@@ -9,7 +9,7 @@ export const TableCell = createComponent<'td', CellOptions>((props) => {
     onClick,
     csx,
     role = 'cell',
-    density = 'regular',
+    density = 'variable',
```

Comment:
**PS:** I'm just setting every one to have `variable` as the default density and in the next PR I will remove this prop and let its style inside the baseline style.

Reply:
I want to avoid in this PR a lot changes related to the `density` prop